Core/Core.Platform.Android/Utils/PlatformBridge/AndroidUserPreferences.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using Android.Content;
+
+namespace AeroGear.Mobile.Core.Utils
+{
+    public class AndroidUserPreferences : IUserPreferences
```

Comment:
IIRC, this is similar to the "IStorageManager" interface: https://github.com/aerogear/aerogear-xamarin-sdk/blob/master/Core/Core/Storage/IStorageManager.cs. Can we unify them?

Reply:
agree with @wei-lee  on that